HTML/mockup-mogo/js/task7.js

```diff
@@ -0,0 +1,66 @@
+import {promiseElements} from './elements.js';
+
+export const initSlider = () => {
+  promiseElements.then((data) => {
+    const {
+      opinionsPartFirst,
+      opinionsPartSecond,
+      buttonLeft,
+      buttonRight,
+    } = data;
+    let curIndexFirst = 0;
+    let curIndexSecond = 0;
+
+    buttonLeft.forEach((el, ind) => {
+      el.addEventListener('click', () => {
+        if (ind === 0) {
+          curIndexFirst = opinionChange(
+              opinionsPartFirst,
+              curIndexFirst,
+              'Left',
+              'opinionsPartFirst'
+          );
+        } else {
+          curIndexSecond = opinionChange(
+              opinionsPartSecond,
+              curIndexSecond,
+              'Left',
+              'opinionsPartSecond'
+          );
+        }
+      });
+    });
+
+    buttonRight.forEach((el, ind) => {
+      el.addEventListener('click', () => {
+        if (ind === 0) {
+          curIndexFirst = opinionChange(
```

Comment:
not very good way doing things, as you will need to create variable with index for every slider